config/networks.json

```diff
@@ -787,6 +787,26 @@
     "deployedWithSolcVersion": "0.8.17",
     "create3Factory": "0x93FEC2C00BfE902F733B57c5a6CeeD7CD1384AE1"
   },
+  "plasma": {
+    "name": "plasma",
+    "chainId": 9745,
+    "nativeAddress": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+    "nativeCurrency": "WXPL",
```

Comment:
not only "XPL" ?

Reply:
XPL，I think.
The BE is using 0x000 as native token address, do we need to change it?

Reply:
@mirooon 
yes, XPL is correct. Good catch. I've changed it.